### PR TITLE
feat(feedback): revert feedback changes and add onModalOpenChange prop

### PR DESCRIPTION
**Pull request by Satyampc**
**Before submitting a pull request,** please make sure the following is done:

1. Fork [the repository](https://github.com/availity/availity-react) and create your branch from `master`.
2. Run `yarn` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
5. Make sure your code passed the conventional commits check. Read more about [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)

### CODE REVIEW

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://6172c9a3da352329e257c0da--condescending-leavitt-66e607.netlify.app

File: packages/select/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@availity/select",
-  "version": "1.0.0",
```
Comment (chrishavekost) [inline packages/select/package.json]: Can we remove this? `master` already has this change so there's no need to duplicate it. Or if you are running into some conflict you should pull in the latest changes from `master` to your branch and have a merge commit for it

File: packages/select/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.0.1](https://github.com/Availity/availity-react/compare/@availity/select@1.0.0...@availity/select@1.0.1) (2021-10-19)
```
Comment (chrishavekost) [inline packages/select/CHANGELOG.md]: Same comment: Can we remove this? `master` already has this change so there's no need to duplicate it. Or if you are running into some conflict you should pull in the latest changes from `master` to your branch and have a merge commit for it

**Review by chrishavekost — CHANGES_REQUESTED** [review]

**Comment by chrishavekost** [comment]
For others wondering about this PR: @LauRoxx brought up a few unintended consequences from a recent feedback PR, and it was determined it would be easier to revert those for now until a larger overhaul of the feedback package could be worked on.

File: packages/feedback/types/Feedback.d.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ export interface FeedbackProps extends DropdownProps {
   formProps?: { [key: string]: any };
   prompt?: React.ReactType;
   onFeedbackSent?: (feedback?: { [key: string]: any }) => void;
+  onModalOpenChange?: (feedback?: { [key: string]: any }) => void;
```
Comment (danielpickett) [inline packages/feedback/types/Feedback.d.ts]:
```suggestion
  onModalOpenChange?: (isOpen: boolean) => void;
```

**Review by danielpickett — COMMENTED** [review]
Aside from Chris' comments, the only thing I see is my suggestion on the TS type declaration for `onModalOpenChange`.

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://6172e4b3935e4469ad946366--condescending-leavitt-66e607.netlify.app

File: packages/select/CHANGELOG.md (hunk 2 above)
Comment (Satyampc) [reply packages/select/CHANGELOG.md]: Resolved the conflicts and changes

**Review by danielpickett — DISMISSED** [review]

**Review by danielpickett — APPROVED** [review]

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://6172e6f23a000751cf98616e--condescending-leavitt-66e607.netlify.app

**Pull request merged**